pkg/ddl/tests/metadatalock/mdl_test.go

```diff
@@ -997,6 +1001,104 @@ func TestMDLPreparePlanCacheExecute2(t *testing.T) {
 	tk.MustExec("admin check table t")
 }
 
+func TestMDLPreparePlanCacheExecuteInsert(t *testing.T) {
```

Comment:
This test is quite complicated, could you add some comments?

Reply:
okey